configure

```diff
@@ -14990,7 +15088,7 @@ else
     We can't simply define LARGE_OFF_T to be 9223372036854775807,
     since some C++ compilers masquerading as C compilers
     incorrectly reject 9223372036854775807.  */
-#define LARGE_OFF_T (((off_t) 1 << 62) - 1 + ((off_t) 1 << 62))
+#define LARGE_OFF_T ((((off_t) 1 << 31) << 31) - 1 + (((off_t) 1 << 31) << 31))
```

Comment:
For some reason `autoreconf -i` has changed some irrelevant lines in configure. To be honest, I don't care that much (after all, we can always re-generate the configure script), but this might lead to merge conflicts.

Reply:
You're running Debian, I presume? Or a derivative like Ubuntu. Debian carries a patch to autoconf that caues that. I recommend installing the vanilla version from https://www.gnu.org/software/autoconf/#downloading to avoid that. Although as you said, it's not a big deal.

Reply:
> You're running Debian, I presume? 

Not exactly. My daily driver is ArchLinux, which provides autoconf 2.71, which in its turn is not suitable for postgres (it requires 2.69). So I had to run autoreconf in a docker container with ubuntu:latest.

> I recommend installing the vanilla version from https://www.gnu.org/software/autoconf/#downloading to avoid that. 

Good to know, thanks. I wonder if there's an off-the-shelf docker image with a vanilla version...